index.html

```diff
@@ -98,7 +114,7 @@ <h2 class="title-post-2">Profesional Art Printing Data</h2>
               <li><button class="p-lang-2">Ruby</button></li>
             </ul>
           </div>
-          <button class="link-btn-2 fill-h">See Project</button>
```

Comment:
- Kindly make sure each project has the `see project `button in your desktop version only one project have the seen button please fix that so   when someone will click on that see project button it  will show the popup with all detail of your project :+1: 

![image](https://user-images.githubusercontent.com/95297251/190877313-f369979b-2782-4d77-ba70-55b562744640.png)